docs/data/sdks/sdk-quickstart.md

```diff
@@ -572,56 +555,130 @@ Use this guide to get started with the Amplitude SDKs. Choose your target platfo
 
     Learn more available functionalities in [Android SDK](../android-kotlin/).
 
-=== "iOS"
+    --8<-- "includes/sdk-quickstart/quickstart-enforce-event-schema-intro.md"
+
+    === "Kotlin"
+
+        ```kotlin
+
+        ampli.load()
+
+        ampli.yourEventType(
+            stringProp = "Strongly typed property",
+            booleanProp = true
+        )
+
+        ```
+    === "Java"
+
+        ```java
+        Ampli.getInstance().load();
+
+        Ampli.getInstance().yourEventType(
+            YourEventType.builder()
+                .stringProp("Strongly typed property")
+                .booleanProp(true)
+                .build()
+        );
+        ```
+
+    Learn more about [Ampli Android](../android-kotlin/ampli/).
 
+=== "iOS-Legacy"
     The iOS SDK lets you send events to Amplitude. See the full documentation at [iOS SDK](../ios/)
 
+    !!!deprecated "Legacy SDK"
+        This is a legacy SDK and will only receive bug fixes until deprecation. Upgrade to the latest [iOS Swift SDK](../ios-swift/) which supports plugins and more. See the [Migration Guide](../ios-swift/migration/) for more help. 
+        
+        Please note that the latest iOS Swift SDK is **NOT** compatible with Objective-C projects. Use this SDK if your project requires compatibility with Objective-C. 
```

Comment:
I feel this is too aggressive given the current state of the SDKs. "Legacy" sounds like nobody should use this SDK, and that we have a full fledged replacement available. I think it will be confusing to users to choose from "Legacy" and "Beta" with no "Current GA" version.

Maybe we can make the language less scary:

> This is our time tested iOS SDK, however we have a new version in beta that we recommend for all customers using Swift. The latest [iOS Swift SDK](../ios-swift/) has additional features such as plugins and more. See the [Migration Guide](../ios-swift/migration/) for more help. 
>         
>         Please note that the latest iOS Swift SDK is **NOT** compatible with Objective-C projects. Use this SDK if your project requires compatibility with Objective-C.